memo.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader' if development?
+require 'json'
+require 'securerandom'
+
+get '/memos' do
+  @memos = read_memos_table
```

Comment:
```suggestion
  @memos = read_memos
```

https://bootcamp.fjord.jp/pages/readable-variable-name を参考に〜。

Reply:
ハッシュの命名に悩んでいますが、`*_table`や`*_hash`や`{key}_to_{value}`などで変数名を見た時にハッシュと分かるような命名をした方がいいのかと思っていましたが名前が長くなりますし、配列と同じくデータの複数形の命名の方が分かりやすいという認識でよろしかったでしょうか？
e08b286 にて対応しました！

Reply:
ハッシュは「ただの複数形」として命名されるケースもよくあるので、memosでも問題ないと思いますー。